src/db/setups/users.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+const Users = require('../models/users');
+
+Users.sync({ alter: true });
```

Comment:
Consider wrapping `Users.sync({ alter: true });` in an asynchronous function, to handle possible errors and ensure proper execution. Example: `async function syncUsers() { await Users.sync({ alter: true }); }`